decidim-process-extended/app/serializers/decidim/participatory_processes/participatory_process_serializer.rb

```diff
@@ -61,20 +84,52 @@ def serialize
           debates_num_debates: debates.count,
           # Related Resources: Assemblies
           has_related_assembly: related_assembly.present?,
-          related_assembly_name_ca: related_assembly&.title.try(:[], "ca")
+          related_assembly_name_ca: related_assembly&.title.try(:[], "ca"),
+          attachments: {
+            attachment_collections: serialize_attachment_collections,
+            files: serialize_attachments
+          },
         }
       end
 
       private
 
       attr_reader :process
 
-      def url
-        Decidim::ResourceLocatorPresenter.new(process).url&.split("?")&.first
+      def serialize_attachment_collections
+        return unless process.attachment_collections.any?
+
+        process.attachment_collections.map do |collection|
+          {
+            id: collection.try(:id),
+            name: collection.try(:name),
+            weight: collection.try(:weight),
+            description: collection.try(:description)
+          }
+        end
+      end
+
+      def serialize_attachments
+        return unless process.attachments.any?
+
+        process.attachments.map do |attachment|
+          {
+            id: attachment.try(:id),
+            title: attachment.try(:title),
+            weight: attachment.try(:weight),
+            description: attachment.try(:description),
+            attachment_collection: {
+              name: attachment.attachment_collection.try(:name),
+              weight: attachment.attachment_collection.try(:weight),
+              description: attachment.attachment_collection.try(:description)
+            },
+            remote_file_url: Decidim::AttachmentPresenter.new(attachment).attachment_file_url
+          }
+        end
       end
 
-      def short_description_ca
-        ActionController::Base.helpers.strip_tags(process.short_description["ca"])
```

Comment:
Why remove and take all translations instead?

Reply:
it has not been removed but rather it has been refactored and now there are two serializers, one for Socrata and another for the processes with an override with the custom fields